docs/snippets/add-or-modify-states.md

```diff
@@ -0,0 +1,30 @@
+# Add or modify states
+
+> This is a **Developer level** doc. If you are unfamiliar with code and resolving potential conflicts, select a [WooExpert or Developer](https://woocommerce.com/customizations/) for assistance. We are unable to provide support for customizations under our [Support Policy](http://www.woocommerce.com/support-policy/).
+
+Add this code to your child theme’s `functions.php` file or via a plugin that allows custom functions to be added, such as the [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin. Avoid adding custom code directly to your parent theme’s functions.php file, as this will be wiped entirely when you update the theme.
+
+Add your own or modify shipping states in WooCommerce.
+
+> **Note**
+> You must replace both instances of XX with your country code. This means each state id in the array must have your two letter country code before the number you assign to the state.
```

Comment:
Should this be plain text rather than a block-quote, as in the original doc? Perhaps we need a better way to add this sort of call-out in a way that is distinct from block-quotes.

```suggestion
> Note: you **must** replace both instances of XX with your country code. This means each state id in the array must have your two letter country code before the number you assign to the state.
```